packages/minter-contracts/ligo/src/fixed_price_sale/README.md

```diff
@@ -0,0 +1,265 @@
+# Fixed Price Sale
+
+The contracts titled `fixed_price_sale_market.mligo` and
+`fixed_price_sale_market_tez.mligo` both contain implementations of a sale of a
+non-fungible token (NFT) at a fixed price. The former contract allows a seller
+to sell an NFT for some fixed price of FA2 tokens. The latter instead allows a
+seller to sell an NFT for some fixed price of tezzies. Although, the general
+design and implementation of this contract is fairly straightforward, someone
+unfamiliar with the Tezos blockchain's inner workings needs to be aware of
+certain key items. These details will discussed in the following section
+[Entrypoints](#entrypoints-section). For now, we shall discuss the storage
+contents of each contract.
+
+## <a name="storage-section"></a> Storage
+
+The next two subsections will list and explain the storage contents of each
+contract. For the sake of concision, we hereby label the contract
+`fixed_price_sale_market.mligo` as `SALE_FA2` and the other
+(`fixed_price_sale_market_tez.mligo`) as `SALE_TEZ`.
+
+### <a name="sale-fa2-section"></a>`SALE_FA2` Storage
+
+The contract's storage is:
+
+``` ocaml
+type storage = {
+    admin: pauseable_admin_storage;
+    sales: (sale_param, nat) big_map;
+}
+```
+
+The first field of this record is `admin`. Its type is:
```

Comment:
Consider replacing admin with one of the new admin modules

Reply:
Is there a particular module you have in mind that would work better here?

Reply:
Well, `simple_admin` would do, but JA wanted multi-admin for some of the contracts, but I am not sure which ones.